### PR TITLE
3343-Convenience-features-Content-update

### PR DESCRIPTION
**Pull request by ramkumar-k-9286**
Addition of Information on the convenience features

#3343

### CODE REVIEW

File: modules/reference/pages/feature/feature-overview.adoc [hunk 1]
```diff
@@ -48,6 +48,24 @@ The runtime contains default configuration settings so that the configuration yo
 Specify the features that you need, along with any additions or overrides to the default settings, in the `server.xml` file.
 For details about the server configuration, see the xref:config/server-configuration-overview.adoc[server configuration overview].
 
+You can enable a whole set of features all at once using the convenience features. Use these convenience features in the feature manager of your server configuration to add support for your Open Liberty server.
```
Comment (dmuelle) [inline modules/reference/pages/feature/feature-overview.adoc]: You can enable a whole set of features all at once using the convenience features
--->
You can enable a whole set of features all at once by specifying one of the convenience features.

Use these convenience features in the feature manager of your server configuration to add support for your Open Liberty server.
--->
Convenience features automatically enable all the individual Open Liberty features that are associated with a particular platform or functional area. For example, the Jakarta EE Platform feature enables all the features that are part of the xref:ROOT:jakarta-ee.adoc[Jakarta EE Platform]. The Jakarta EE Web Profile feature enables only those features that are part of the Jakarta EE Web Profile. A list of all the features that a convenience features enables is provided on the reference page for each convenience feature.

File: modules/reference/pages/feature/feature-overview.adoc [hunk 2]
```diff
@@ -48,6 +48,24 @@ The runtime contains default configuration settings so that the configuration yo
...
+The following are considered as convenience features for Jakarta 9.1:
```
Comment (dmuelle) [inline modules/reference/pages/feature/feature-overview.adoc]: suggested edit- use the definition list syntax:

The following convenience features are available for the for Jakarta EE platform:
feature:jakartaee[display=Jakarta EE Platform]::
This feature enables all features that are associated with the Jakarta EE platform

feature:webProfile[display=Jakarta EE Web Profile]::
This feature enables only features that are associated with the lightweight link:https://jakarta.ee/specifications/webprofile/9/[Jakarta EE Web Profile].

feature: jakartaeeClient[display=Jakarta EE Application Client]::
This feature enables all features that are associated with Jakarta EE link:https://jakarta.ee/specifications/platform/9/jakarta-platform-spec-9.html#a3294[Application Client] functions.

Additionally, the feature:enterpriseBeans[Jakarta Enterprise Beans] convenience feature automatically enables all Open Liberty features that support enterprise beans functions.

File: modules/reference/pages/feature/feature-overview.adoc [hunk 3]
```diff
@@ -48,6 +48,24 @@ The runtime contains default configuration settings so that the configuration yo
...
+* Jakarta EE Platform 9.1 - To add support for Jakarta EE to your Open Liberty server
+* Jakarta EE Web Profile 9.1 - To add support for Jakarta EE to your Open Liberty server
+* Jakarta EE Application Client 9.1 - To configure an application client component
+* EnterpriseBeans-4.0 - No unique function, only enable other features.
+
+Use the convenience features in the feature manager of your server configuration; for example:
```
Comment (dmuelle) [inline modules/reference/pages/feature/feature-overview.adoc]: Enable convenience features by adding them to the `featureManager` element in your  `server.xml` file, as shown in the following example.

**Review by dmuelle — CHANGES_REQUESTED** [review]
looks good, a few suggestions

**Comment by ramkumar-k-9286** [comment]
@dmuelle 

All the review comments specified above have been incorporated.

Proceeding to Merge the PR. Will share the draft link for review.

**Comment by ramkumar-k-9286** [comment]
@dmuelle 

Suggested changes were made.Link to the draft version of the document:

 https://draft-openlibertyio.mybluemix.net/docs/21.0.0.12/reference/feature/feature-overview.html

Regards,
Ramkumar

**Pull request merged**